src/engine/drpc_client.c

```diff
@@ -171,6 +171,8 @@ drpc_notify_ready(void)
 	if (rc != 0)
 		goto out_uri;
 
+	/* TODO: Add secondary URIs */
```

Comment:
As we discussed on skype, you could call the new Cart API crt_self_uri_get_secondary(int secondary_idx, char **uri) to get the secondary uri.